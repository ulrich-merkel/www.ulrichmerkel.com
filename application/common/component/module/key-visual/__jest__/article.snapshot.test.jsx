/* eslint-disable import/no-extraneous-dependencies, func-names */
import React from 'react';
import renderer from 'react-test-renderer';
import ComponentToBeTested from './../article';

describe('component/module/key-visual/article', function () {
    describe('Snapshot', function () {
        it('should render correctly', function () {
            const tree = renderer.create(
                <ComponentToBeTested
                    headline='headline'
                    lead='lead'
                />
            ).toJSON();
            expect(tree).toMatchSnapshot();
        });
        it('should render null if no content is provided', function () {
            const tree = renderer.create(
                <ComponentToBeTested />
            ).toJSON();
            expect(tree).toMatchSnapshot();
        });
    });
});
