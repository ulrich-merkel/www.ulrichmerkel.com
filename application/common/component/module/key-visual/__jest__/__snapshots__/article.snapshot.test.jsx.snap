exports[`component/module/key-visual/article Snapshot should render correctly 1`] = `
<article
  className="m-key-visual__text">
  <h3
    className="is-centered c-type--h3 m-key-visual__headline"
    itemProp="headline">
    <span
      className="m-key-visual__boxed">
      headline
    </span>
  </h3>
  <p
    className="c-type--p m-key-visual__lead"
    itemProp="description">
    <span
      className="m-key-visual__boxed">
      lead
    </span>
  </p>
</article>
`;

exports[`component/module/key-visual/article Snapshot should render null if no content is provided 1`] = `null`;
